setup.cfg

```diff
@@ -39,7 +39,7 @@ gmpy =
     gmpy2>=2.1.0a4; platform_python_implementation!="PyPy"
 docs = sphinx
 [pycodestyle]
-select = E101,W191,W291,W293,E111,E112,E113,W292,W391
+select = E10,E11,E13,E703,E71,E9,W1,W2,W3,W6
```

Comment:
No, please don't do this.  Feel free to enable more checks (already fixed;)), but lets do this explicitly.  I don't think we want to break our CI if flake8 people just add some new E9XY error with the new release.

Reply:
The idea behind linters is that we set up guardrails so that if contributors break rules that the codebase already passes those violations can be discussed in code review.

Reply:
> if contributors break rules that the codebase already passes

Exactly.  Not something that just coming with a new release of the linter.

Reply:
Another possibility is to pin the version of the linter.

Reply:
New release might fix bugs in existing checks.  Probably, we want this and it shouldn't be too annoying, i.e. break too often and too much.  (Such fixes usually cover some corner cases and so on...)  But if it will - we can consider pinning.